### ISSUE TITLE
Contact Picture Size

### ISSUE THREAD

**Issue by ghost** [owncloud/apps#454]
Can you please add a possibility to set the size of the Contact Picture? I think its only hardcoded to 200x200 Pixel at this time. I want bigger images, since i would sync OwnCloud Contacts to my mobile. With this, the Contact Picture works as a CallerID and its displayed when the contact calls me. With higher resolution phone displays, it would be a good idea, to support this larger resolutions.

The only thing you should keep in mind is, if the image is bigger than 48kb (filesize) and the Contacts are synced to a Phone via EAS (Exchange Active Sync), this is the max supported resolution according to the Microsoft Specifictaions. If i use Z-Push to Sync via EAS, i can extend this to higher value, but this is not specified.

Could you please add such a feature?

**Comment by frode** [comment]
I Agree, please let us choose and set the 200x200 as the default size.

**Comment by FrederikLauber** [comment]
@tanghus 
Maybe I misunderstand but you said that if i use a high res picture as a contact on my phone, this high res picture will be synced to owncloud but the web page will only show 200x200 pixels?
This seems to be incorrect. I added a high res picture to my contacts, then synced to owncloud. Afterwards i had to wipe my phone and synced my contacts back to my phone.
The result was very ugly contact pictures. So somewhere on the way, without touching the web client, I ended up with 200x200 pictures....
